### DISCUSSION TITLE
Netdiscovery implement Net:nslookup

### DISCUSSION THREAD

**Discussion by jpnr79** [glpi-project/glpi-agent#406] [Ideas]
### Your idea

Some net discovery, just give  dnshostname the IP and the IP is the port and IP is hostname or device name, if have a step to discovery name because ar, netbios and others fail, make the nslookup to retrive the name, I think it is not so dificult

**Comment by g-bougard** [comment]
I don't understand what you mean by "dnshostname the IP and the IP is the port and IP is hostname or device name". Have you any sample output ?
You can run `glpi-netdiscovery --timeout 1 --host A.B.C.D` to obtain an output sample.

**Comment by jpnr79** [comment]
this is the output
[image: image.png]
dont have the host name

with nslookup i have fqdn
[image: image.png]

Guillaume Bougard ***@***.***> escreveu no dia segunda,
29/05/2023 à(s) 09:58:

> I don't understand what you mean by "dnshostname the IP and the IP is the
> port and IP is hostname or device name". Have you any sample output ?
> You can run glpi-netdiscovery --timeout 1 --host A.B.C.D to obtain an
> output sample.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/glpi-project/glpi-agent/discussions/406#discussioncomment-6027773>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGJFAZ5PGSWI75O7E7LL353XIRQLXANCNFSM6AAAAAAYQQ7SCA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Pedro Rocha
Faro
email: ***@***.*** ***@***.***>
Telf.: +351 935 478 010 <+351%20935%20478%20010>
linkedin: Pedro Rocha <http://www.linkedin.com/in/pedro-rocha79>

**Reply by g-bougard** [reply]
Images are not displayed. Please answer from the site, not by email.
